matrimony_admin/views.py

```
@@ -1,30 +1,33 @@
-from U_auth.models import costume_user
+# Django ORM and database functions
 from django.db.models import Sum, Count, Q
-from django.db.models.functions import TruncDay
-from subscription.models import Payment
-from django.utils import timezone
-from datetime import timedelta
+from django.db.models.functions import TruncDay, TruncMonth
 
+# Project-specific imports (models, forms, and permissions)
+from .models import BlockedUserInfo, Add_expense
+from .forms import AdminLoginForm, AdminProfileForm, NotificationDetailsForm, AdminProfileEditForm, AddExpenseForm
```

Ruff

`.forms.AdminProfileForm` imported but unused

Remove unused import

(F401)